### PR TITLE
July 2024 newsletter

### PR DESCRIPTION
**Pull request by cticenhour**
Refs #11447 

@idaholab/moose-ccb Please add content that you have added this month. Please also tag those who are collaborators or external contributors who you have reviewed PRs for!

### CODE REVIEW

File: modules/doc/content/newsletter/2024/2024_07.md
Comment (cticenhour) [inline modules/doc/content/newsletter/2024/2024_07.md:67]:
```suggestion
```

**Comment by GiudGiud** [comment]
I ll have time today to contribute

**Comment by cticenhour** [comment]
Ok! 👍 I'll hold off one more day.

**Comment by GiudGiud** [comment]
they are in now

**Review by cticenhour — COMMENTED** [review]
Thanks @GiudGiud! I approve your commit. Need your approval to merge, if you are willing to give it!

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**
